### PR TITLE
Style Core Elements

### PR DESCRIPTION
**Pull request by John-Arboleda**
Awesome Books is a basic website that allows users to add/remove books from a list. This is achieved by using JavaScript objects and arrays. The DOM is modified dynamically and adds basic events.

In this pull request, a medium-fidelity wireframe was created with CSS.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+.button {
+  align-self: flex-end;
+  margin-right: 50px;
+  background-color: white;
+  width: 65px;
+  border: black 2px solid;
+  box-shadow: 4px 2px 2px black;
+}
```
Comment (Dipeshtwis) [inline style.css]: - [ ] The `add` button should be on the right side below the input field. Please update to stick with the design.

### Your design

Screenshot from 2021-11-25 00-00-22

### Original design

Screenshot from 2021-11-25 00-00-35

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @John-Arboleda and @franklinben23, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (henatan99) [reply style.css]: This issue has been addressed well. Great job! :+1: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @John-Arboleda , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**